modules/openapi-generator/src/test/java/org/openapitools/codegen/DefaultGeneratorTest.java

```diff
@@ -662,6 +662,30 @@ public void testHandlesTrailingSlashInServers() {
         Assert.assertEquals(servers.get(1).url, "http://trailingshlash.io:80/v1");
         Assert.assertEquals(servers.get(2).url, "http://notrailingslash.io:80/v2");
     }
+    
+    @Test
+    public void testHandlesRelativeUrlsInServers() {
```

Comment:
method `configureGeneratorProperties` is package protected, you didn't need to write this whole test with setup of new yaml file